home/templates/home/item.html

```diff
@@ -22,36 +22,4 @@ <h4 class="media-heading"><a href="view">{{ post.title }}</a></h4>
     </div>
```

Comment:
Do we need this view at all anymore? I don't think you can actually get to it anymore.